DT.iOS.DatePickerDialog/DatePickerDialog.cs

```diff
@@ -38,40 +41,44 @@ public class DatePickerDialog : UIView
         private NSObject? _orientationChangeObserver;
 
         public DatePickerDialog(UIColor? textColor = null, UIColor? buttonColor = null, UIFont? font = null, NSLocale? locale = null,
-            bool showCancelButton = true)
+            bool showCancelButton = true, bool useLocalizedButtons = false)
         {
             _textColor = textColor ?? Colors.Text();
             _buttonColor = buttonColor ?? Colors.Accent();
             _font = font ?? UIFont.BoldSystemFontOfSize(15);
             _showCancelButton = showCancelButton;
+            _useLocalizedButtons = useLocalizedButtons;
             _locale = locale;
             SetupView();
         }
 
         public void Show(string title, Action<DateTime> callback, DateTime minimumDate, DateTime maximumDate, Action? cancelCallback = null)
-        {
-            Show(title, doneButtonTitle: "Done", cancelButtonTitle: "Cancel", datePickerMode: UIDatePickerMode.DateAndTime, callback: callback, defaultDate: DateTime.Now, maximumDate: maximumDate, minimumDate: minimumDate, cancelCallback: cancelCallback);
-        }
+            => Show(title, doneButtonTitle: kDone, cancelButtonTitle: kCancel, datePickerMode: UIDatePickerMode.DateAndTime, callback: callback, defaultDate: DateTime.Now, maximumDate: maximumDate, minimumDate: minimumDate, cancelCallback: cancelCallback);
 
         public void Show(string title, Action<DateTime> callback, UIDatePickerMode datePickerMode = UIDatePickerMode.DateAndTime, Action? cancelCallback = null)
-        {
-            Show(title, doneButtonTitle: "Done", cancelButtonTitle: "Cancel", datePickerMode: datePickerMode, callback: callback, defaultDate: DateTime.Now, cancelCallback: cancelCallback);
-        }
+            => Show(title, doneButtonTitle: kDone, cancelButtonTitle: kCancel, datePickerMode: datePickerMode, callback: callback, defaultDate: DateTime.Now, cancelCallback: cancelCallback);
```

Comment:
Thanks for this, I did not want to do that syntax refactoring in the recent release (did not have much time to get it right) so this is definitely welcome!

Reply:
I did it out of habit😀